docker/php-console/entrypoint.sh

```diff
@@ -3,13 +3,8 @@ set -e
 
 setup
 
-# Wait for composer vendor
 if [ ! -f vendor/autoload.php ]; then
-    echo "Run 'composer install' command to continue."
+	su "$(stat -c '%U' composer.json)" -c 'composer install'
```

Comment:
@Soullivaneuh I reimplemented this, this time using the composer binary from the php-console image. Looks good?